mcfix_purchase/models/purchase.py

```diff
@@ -137,6 +137,17 @@ def _check_company_id(self):
 class PurchaseOrderLine(models.Model):
     _inherit = 'purchase.order.line'
 
+    company_id = fields.Many2one(
+        'res.company', related=[], compute='_compute_line_company',
+        string='Company', store=True, readonly=True)
```

Comment:
you shouldn't put the unchanged attributes

Reply:
It is changed - it used to be related, it is now computed....

Reply:
yeah, but you don't need to put again 'res.company', string, store, ...